src/logius/github.js

```diff
@@ -0,0 +1,53 @@
+// @ts-check
+
+import { showWarning } from "../core/utils.js";
+/**
+ * this module fixes some peculiarities in core/github for the Dutch context
```

Comment:
I wonder if we can discuss this as a fix to the Core module? I wasn't able to figure out what you were trying to do here, but we can probably fix whatever the issue is in core to make it work without requiring this module?